docs/book/src/quick-start.md

```diff
@@ -7,9 +7,19 @@ This Quick Start guide will cover:
 - [Running locally](#test-it-out-locally)
 - [Running in-cluster](#run-it-on-the-cluster)
 
+## Prerequisites
+
+- [go](https://golang.org/dl/) version v1.12+.
+- [docker](https://docs.docker.com/install/) version 17.03+.
+  - Alternatively [podman](https://github.com/containers/libpod/blob/master/install.md) `v1.2.0+` or [buildah](https://github.com/containers/buildah/blob/master/install.md) `v1.7+`
```

Comment:
we don't provide podman and buildah instructions, so I'm hesitant to include these specifically.  Perhaps `(or some other way to build and run containers)` or something similar.

Reply:
Done